Makefile

```diff
@@ -36,7 +36,7 @@ DOCKER_BUILD ?= $(DOCKER) run --rm -v $(DIR):$(BUILDMNT) -w $(BUILDMNT) $(BUILD_
 all: container
 
 container:
-	$(DOCKER_BUILD) 'CGO_ENABLED=0 go build'
+	$(DOCKER_BUILD) 'CGO_ENABLED=0 go build -mod=vendor'
```

Comment:
Why do we need vendor?

Reply:
-mod=vendor will directly use the local vendor to compile, so compile will more faster~